pkg/policies/branch/branch.go

```diff
@@ -319,6 +376,14 @@ func fix(ctx context.Context, rep repositories, c *github.Client,
 					}
 					pr.RequiredPullRequestReviews = rq
 				}
+				if mc.RequireUpToDateBranch || len(mc.RequireStatusChecks) > 0 {
```

Comment:
Change to `if len(mc.RequireStatusChecks) > 0`

Reply:
Fixed in 754144bd3f9e24f3554793450990d44d4116b98f